version.py

```diff
@@ -34,7 +34,7 @@
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
 
-API_FEATURE_LEVEL = 380
+API_FEATURE_LEVEL = 381  # bumped 2024-04-27 to add reminders endpoint
```

Comment:
You don't need to do this anymore; `tools/create-api-changelog` is the better mechanism since it means one doesn't need to renumber.

Reply:
That's nice! Used that.